webapp/src/main/webapp/news-list-view/components/NewsListView.vue

```diff
@@ -0,0 +1,118 @@
+<!--
+Copyright (C) 2021 eXo Platform SAS.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+<template>
+  <v-app class="news-list-view-app position-relative">
+    <v-hover v-slot="{ hover }">
+      <v-card flat class="list-view-card">
+        <v-card-text class="pa-0">
+          <news-settings v-if="$root.saveSettingsURL && viewTemplate === 'NewsLatest'" :hover="hover" />
```

Comment:
why we need testing on viewTemplate ?

Reply:
to display top news settings button and see all only for latest news